elastic/kibana
# fix export response

## Summary

This PR is to fix the broken export functionality.

Steps to verify:

Click on the `export button` for those exportable timeline / templates and see if it completed successfully.
To get some sample Elastic templates, you can download, rename it to `.ndjson` and import this file:

[index copy.txt](https://github.com/elastic/kibana/files/4860204/index.copy.txt)

- timelines (exportable)
- Elastic templates (Not exportable)
- Custom templates (exportable)


<img width="1615" alt="Screenshot 2020-07-01 at 17 02 50" src="https://user-images.githubusercontent.com/6295984/86266177-ba2d5400-bbbc-11ea-8318-48814b13511d.png">
<img width="1605" alt="Screenshot 2020-07-01 at 17 02 37" src="https://user-images.githubusercontent.com/6295984/86266189-bc8fae00-bbbc-11ea-9b71-8d324934dad6.png">


### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
